src/Groups/pcgroup.jl

```diff
@@ -199,6 +265,31 @@ function set_conjugate!(c::Collector{T}, j::Int, i::Int, rhs::Vector{Pair{Int, T
   end
 end
 
+"""
+    get_conjugate(c::Collector{T}, j::Int, i::Int) where T <: IntegerUnion
+
+Get the `Vector{Pair{Int, T}}` that describes the conjugate of the `j`-th
+generator of `c` by the `i`-th generator of `c`, for `i < j`.
+
+# Examples
+```jldoctest
+julia> c = collector(2, Int);
+
+julia> set_relative_orders!(c, [2, 3])
+
+julia> set_conjugate!(c, 2, 1, [2 => 2])
+
+julia> get_conjugate(c, 2, 1)
+1-element Vector{Pair{Int64, Int64}}:
+ 2 => 2
+```
+"""
+function get_conjugate(c::Collector{T}, j::Int, i::Int) where T <: IntegerUnion
+  @req 0 < i <= c.ngens "the collector has only $(c.ngens) generators not $i"
+  @req i < j "only for i < j, but i = $i, j = $j"
+  return c.conjugates[i,j]
```

Comment:
This will fail if the entries is trivial, ie. for abelian groups:
```
G = abelian_group(PcGroup, [2,3]);
c = collector(G);
julia> c.conjugates
2×2 Matrix{Vector{Pair{Int64, ZZRingElem}}}:
 #undef  #undef
 #undef  #undef
```
so every access (`get_conjugate`) will fail

Reply:
Yes, thanks. I have fixed this.